signingscript/sign.py

```diff
@@ -46,7 +46,7 @@
 
 
 # task_cert_type {{{1
-def task_cert_type(task):
+def task_cert_type(context):
     """Extract task certificate type.
 
     Args:
```

Comment:
Thanks!

Let's update the args in the docstring to match.

Should we specify `project:releng` as the prefix and use it for formats too? That means we'd want to update `signingscript.task.task_signing_formats`.

This changes the public api, so even though we don't have downstream consumers, we're probably looking at a 5.0.0 release for this.

Reply:
Good idea. I made the changes. While we're at major bumping the version, I also moved `task_cert_type()` to `task.py`